src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -160,6 +161,20 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
         return response;
     }
 
+    protected String getDocumentTypeLabel(SscsCaseData caseData, DocumentType documentType, String documentTypeLabel) {
```

Comment:
What is the context of this method? Can it be crafted in such a way to be more easily expanded to the future post hearing types (if required)?

Reply:
A separate method is created, if in future the document name to be modified then only this method impacted. No other changes required.

Reply:
This method will need to be fully refactored on the next Post Hearing journey. This is acceptable as that has been accounted for in the scoring for those tickets.